docs/concepts/multi-chain-proposals.md

```diff
@@ -0,0 +1,693 @@
+It is important to note that the code provided here is a general template and may require modification to suit the specific requirements of the target chain.
+
+## Overview
+
+Of the 24 chains where Uniswap V3 is deployed there are 10 chains that are OP Stack based and use an `L1CrossDomainMessenger` contract for bridging proposals.
+There are 6 chains that use Wormhole for bridging proposals.
+The remaining 8 chains use a variety of other methods for bridging proposals, all described below.
+
+The table below contains links to the various proposal methods for each chain.
+
+| Proposal Method                                       | Chain(s)                                                                             |
+| ----------------------------------------------------- | ------------------------------------------------------------------------------------ |
+| [Arbitrum - Aliased Timelock](#arbitrum)              | Arbitrum                                                                             |
+| [Avalanche - OmnichainGovernanceExecutor](#avalanche) | Avalanche                                                                            |
+| [Filecoin - TODO](#filecoin)                          | Filecoin                                                                             |
+| [Op Stack - CrossChainAccount](#op-stack)             | Base, Blast, Boba, Linea, Manta Pacific, Mantle, Optimism, Redstone, WorldCoin, Zora |
+| [Polygon - FxChild](#polygon)                         | Polygon                                                                              |
+| [Polygon zkEVM - TODO](#polygon-zkevm)                | Polygon zkEVM                                                                        |
+| [Scroll - CrossChainAccount](#scroll)                 | Scroll                                                                               |
+| [Taiko - InvokableAccount](#taiko)                    | Taiko                                                                                |
+| [Wormhole](#wormhole)                                 | BNB Chain, Celo, Gnosis Chain, Moonbeam, Rootstock, Sei                              |
+| [ZkSync Era - Aliased Timelock](#zksync-era)          | ZkSync Era                                                                           |
+
+## Arbitrum
+
+Arbitrum uses an approach where the owner of the V3Factory is a special aliased address (offset by the value 0x1111000000000000000000000000000000001111) that (when the offset is subtracted away) is the L1 address of the Uniswap DAO Timelock contract. A Solidity code example for sending a proposal to Arbitrum is shown below.
+
+Proposal calldata on mainnet intended for Arbitrum should be forwarded through a call to `createRetryableTicket` on the Arbitrum Inbox contract.
+The call would have wrapped calldata for a Uniswap contract function call that would effect the change. An example:
+
+```
```

Comment:
did you explore linking out to these instead?

Reply:
I did this like other parts of the docs repo.. put the code in the doc, but also provided github links to the source files themselves: b96e51a41d6553207a2462e9958acda5bd2857ab